Libraries/Text/Text/RCTTextView.m

```diff
@@ -104,7 +104,12 @@ - (NSString *)description
   NSString *superDescription = super.description;
   NSRange semicolonRange = [superDescription rangeOfString:@";"];
   NSString *replacement = [NSString stringWithFormat:@"; reactTag: %@; text: %@", self.reactTag, _textStorage.string];
-  return [superDescription stringByReplacingCharactersInRange:semicolonRange withString:replacement];
+  // TODO(macOS GH#774): super.description isn't guaranteed to have a semicolon in it on macOS
+  if (semicolonRange.location == NSNotFound) {
+    return [superDescription stringByAppendingString:replacement];
+  } else {
+    return [superDescription stringByReplacingCharactersInRange:semicolonRange withString:replacement];
+  }
```

Comment:
ending TODO tag?

Reply:
Adding on- can you fully enclose the edits in brackets to make it crystal clear where multiline changes start and stop?
`// [TODO ...`
`...`
`/// TODO ... ]`

Reply:
Done with 00f6213.